kubernetes/helm/ckeditor-cs-stack/README.md

```diff
@@ -0,0 +1,51 @@
+# CKEditor Collaboration Server On-Premise stack Helm chart
+
+>:warning: **Not for production purposes!** This configuration is insecure, not persistent and its purpose is testing only.
+
+Helm chart for fast deployment of CKEditor Cloud Services in Kubernetes with MySQL database and redis cluster.
+
+## Getting started
+
+Download repository:
+```sh
+git clone git@github.com:cksource/ckeditor-cs-on-premises-infrastructure.git
+cd ckeditor-cs-on-premises-infrastructure/kubernetes/helm/ckeditor-cs-stack 
+```
+
+Create imagePullSecret for CKEditor container registry
+```sh
+# Your container registry username
+REGISTRY_USERNAME=
+# Your container registry authentication token
+REGISTRY_TOKEN=
+
+kubectl create secret docker-registry docker-cke-cs-com --docker-username $REGISTRY_USERNAME --docker-password $REGISTRY_TOKEN --docker-server https://docker.cke-cs.com
+```
+
+## Installation
+
+Update `values.yaml` file with your ingress hostname and licence key. Next install chart in your kubernetes cluster:
+
+```sh
+helm dependency build
+helm install ckeditor-cs-stack .
+```
+
+## Verify installation
```

Comment:
Container restart and restarts 

![](https://user-images.githubusercontent.com/1795926/117773352-7b364900-b238-11eb-998d-9d81864d3576.png)

Reply:
```
{"level":50,"time":"2021-05-11T07:08:33.018Z","msg":"getaddrinfo ENOTFOUND ckeditor-cs-mysql","data":{"name":"Error","message":"getaddrinfo ENOTFOUND ckeditor-cs-mysql","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","hostname":"ckeditor-cs-mysql","fatal":true}}
```